src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -101,17 +106,28 @@ class BaseCreateCourseForm extends React.Component {
       <div className="create-course-form">
         <h2>Create New Course</h2>
         <hr />
-        <form onSubmit={handleSubmit}>
+        <form onSubmit={handleSubmit} noValidate>
```

Comment:
why is validation being removed when the form is submitted?

Reply:
+1 to this? Validation to the form is a must-have.

Reply:
well this is essentially the point of the PR....this removes the HTML(browser based) validation....if we don't do this then the browser will keep on showing its own tooltips on error fields....